plugins/CoreAdminHome/OptOutManager.php

```diff
@@ -248,13 +613,13 @@ private function optOutStyling()
             }
         }
 
-        if ($cssfontsize && (preg_match("/^[0-9]+[\.]?[0-9]*(px|pt|em|rem|%)$/", $cssfontsize))) {
+        if ($cssfontsize && (preg_match("/^[0-9]+[.]?[0-9]*(px|pt|em|rem|%)$/", $cssfontsize))) {
             $cssbody .= 'font-size: ' . $cssfontsize . '; ';
         } else if ($cssfontsize) {
             throw new \Exception("The URL parameter fontSize value of '$cssfontsize' is not valid. Expected value is for example '15pt', '1.2em' or '13px'.\n");
         }
 
-        if ($cssfontfamily && (preg_match('/^[a-zA-Z0-9-\ ,\'"]+$/', $cssfontfamily))) {
+        if ($cssfontfamily && (preg_match('/^[a-zA-Z0-9-,\'"]+$/', $cssfontfamily))) {
```

Comment:
any reason for disallowing spaces here? won't this break when sending a font family like `'Helvetica Neue'`

Reply:
Good spot :+1: PHPStorm incorrectly flagged this as a redundant escape character and I naively applied the adjustment :man_facepalming:. I'll add a `@noinspection RegExpRedundantEscape` annotation to suppress this.